Tasks/NpmV1/npm.ts

```diff
@@ -45,55 +44,6 @@ async function main(): Promise<void> {
     }
 }
 
-async function _logNpmStartupVariables(packagingLocation: pkgLocationUtils.PackagingLocation) {
-    try {
-        // Log the NPM version
-        let version: string;
-        try {
-            const syncResult: IExecSyncResult = tl.execSync('npm', '--version');
-            if (syncResult.stdout) {
-                version = syncResult.stdout.trim();
-            }
-        } catch (err) {
-            tl.debug(`Unable to get NPM config info. Err:( ${err} )`);
-        }
-
-        // Log the NPM registries
-        const command = tl.getInput(NpmTaskInput.Command) || null;
-        let npmRegistriesAry: INpmRegistry[];
-        const registryUrlAry = [];
-        switch (command) {
-            case NpmCommand.Install:
-            case NpmCommand.Custom:
-                npmRegistriesAry = await npmCustom.getCustomRegistries(packagingLocation);
-                break;
-            case NpmCommand.Publish:
-                npmRegistriesAry = [await npmPublish.getPublishRegistry(packagingLocation)];
-                break;
-        }
-        for (const registry of npmRegistriesAry) {
-            registryUrlAry.push(registry.url);
-        }
-
-        const npmTelem = {
-            'command': command,
-            'verbose': tl.getInput(NpmTaskInput.Verbose) || null,
-            'customRegistry': tl.getInput(NpmTaskInput.CustomRegistry) || null,
-            'customFeed': tl.getInput(NpmTaskInput.CustomFeed) || null,
-            'customEndpoint': tl.getInput(NpmTaskInput.CustomEndpoint) || null,
-            'publishRegistry': tl.getInput(NpmTaskInput.PublishRegistry) || null,
-            'publishFeed': tl.getInput(NpmTaskInput.PublishFeed) || null,
-            'publishEndpoint': tl.getInput(NpmTaskInput.PublishEndpoint) || null,
-            'npmVersion': version,
-            'registries': registryUrlAry
```

Comment:
do we have to remove all of this data or can we just remove the registries? My only concern is breaking workflows requireing these telemetry fields

is it worth keeping npm version/ command ?

Reply:
I felt like after removing everything except the npm version, verbosity, and command it wasn't that useful. I'm not personally aware of anything that depends on this telemetry and I don't see how we could correlate it in a way that would make it useful for targeted investigations of specific failures or to do broad observations about the task because I'm not sure it's possible to correlate this telemetry from the start of the task with any kind of task result